### ISSUE TITLE
help

### ISSUE THREAD

**Issue by songjiaqi1** [kelseyhightower/nocode#4292]
I can not run the code  on the 🚜

**Comment by Hocuri** [comment]
Yes, not being able to run the tractor is a common problem, but it can have lots of different causes.

To narrow down on your issue: Can you give some more details? E.g. type label and license tag of your tractor? Also where the hell you got a yellow tractor from?

Finally, please run ` `  and tell me if this resolves your problem.

(btw sorry for the late answer, I was not busy writing code)

**Comment by Hocuri** [comment]
Ooooooooh, just seen your edit. Sorry, I must have totally gotten this wrong. It wasn't about running the :tractor: itself, it was running the code _on the_ tractor! Good thing we settled this misunderstanding.

Unfortunately, I don't have much experience area of the project myself, but I have some general life advice for you:
- I don't know any specifics about the momentum of the :tractor: and your relative position, but in case it's currently moving towards you at more than, like, 5 m/s, you may want to make a step to the side, shortly before the :tractor: enters the spot where you are currently standing. I have never done this myself and would in no way consider myself an expert in this, but people who have done this usually report what a great feeling it is not to be brutally knocked over by a :tractor:.
- The same is valid in case the tractor is hovering over your current position, even if it's not currently moving towards you. Tractors generally tend not to hover there for very long, so please consider stepping out of the way. Reportedly, even if the tractor is currently moving upwards, this may equally fast change, and the tractor will start falling down. Again, no personal experience, but reportedly it's a great feeling to be in one piece.
